src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_SIDE = 25;
+    private static Random random = new Random();
+    private static final int FIGURE_COUNT = 5;
+    private ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
Overall regarding class properties we start with constants (private static final) and then proceed with other variables.
Also variables for services we rely on are usually private final as we do not plan to pass other object to them and will use same within all class methods
```suggestion
    private static final int MAX_SIDE = 25;
    private static final int FIGURE_COUNT = 5;
    private final Random random = new Random();
    private final ColorSupplier colorSupplier = new ColorSupplier();
```